src/main/java/com/dakar/dakar/resolvers/MutationResolver.java

```diff
@@ -27,6 +27,24 @@ public Journey createJourney(JourneyInput journeyInput) {
         return journeyService.saveJourney(journeyMono).blockFirst();
     }
 
+    /**
+     * Update a Journey
+     *
+     * @param journey
+     * @return The newly updated Journey
+     */
+    public Journey updateJourney(JourneyInput journey) {
+        Journey journeyCreated = new Journey(journey.getId(), journey.getPrice(), journey.getDestination(), "");
+        Mono<Journey> journeyFound = journeyService.findById(journey.getId())
```

Comment:
what happens if it does not found a journey with the given id? Because if it returns an empty journey, it will create a new one?

Reply:
it will return null without creating a new one